packages/frontend/src/components/InboxItem/InboxItemDetail.tsx

```diff
@@ -35,8 +39,20 @@ import styles from './inboxItemDetail.module.css';
  * />
  */
 
-export const InboxItemDetail = ({
-  dialog: {
+export const InboxItemDetail = ({ dialog }: InboxItemDetailProps): JSX.Element => {
+  const { t } = useTranslation();
+  if (!dialog) {
```

Comment:
I think this needs to be moved down passed all useHooks for consistency between each render. Funny React didnt complain about this?